namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-candidate-matching-preprod/resources/ecr.tf

```diff
@@ -0,0 +1,25 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "ecr" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=6.1.0"
+
+  # Repository configuration
+  repo_name = var.namespace
+
+  # OpenID Connect configuration
+  oidc_providers      = ["github"]
+  github_repositories = ["hmpps-candidate-matching-preprod"]
```

Comment:
Is this github repository present? https://github.com/ministryofjustice/hmpps-candidate-matching-preprod

Reply:
Apologies, I've corrected this.